js/main.js

```diff
@@ -283,4 +291,47 @@ $viewEntriesList.addEventListener('click', (e) => {
 
   // Change header from "New Entry" to "Edit Entry"
   $newEntryHeader.textContent = "Edit Entry"
+  $deleteEntryBtn.classList.remove('hidden')
 })
+
+
+/*
+
+    DELETE ENTRY
+
+*/
+
+// Delete Entry => Show Modal
+$deleteEntryBtn.addEventListener('click', () => {
+    $modalBackdrop.classList.remove('hidden')
+})
+
+// Cancel => Hide modal
+$cancelDeleteBtn.addEventListener('click', () => {
+  $modalBackdrop.classList.add('hidden')
+})
+
+// Confirm Delete
+$confirmDeleteBtn.addEventListener('click', () => {
+  const targetId = data.editing.entryId
+
+  // Remove entry from data.entries
+  for (let i = 0; i < data.entries.length; i++) {
+    if (data.entries[i].entryId == targetId) {
+      data.entries.splice(i, 1)
+    }
+  }
+  
+  // Remove entry from DOM
+  for (let domEntry of $viewEntriesList.children) {
+    if (domEntry.getAttribute('data-entry-id') == targetId) {

```

Comment:
Is there a reason you're using `==` instead of `===` here? 
Generally, it's safer to use strict equality to avoid type coersion quirks, javascript can be weird sometimes and might coerce things in counterproductive ways.